src/hooks/useProcessorsApi/useUpdateProcessorApi.ts

```diff
@@ -21,17 +20,23 @@ export function useUpdateProcessorApi(
 } {
   const [processor, setProcessor] = useState<ProcessorResponse>();
   const [error, setError] = useState<unknown>();
-  const [isLoading, setIsLoading] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+  const auth = useAuth();
+
+  const getToken = useCallback(async (): Promise<string> => {
```

Comment:
@kelvah has been moving the token and base URL into the hook. IDK if you want to do the same?

Reply:
And what is different from what is in place here? @manstis this is a hook

Reply:
Since the hook's use is duplicated everywhere we use a custom "API" hook, @kelvah has been moving use of the hook into the custom "API" hook to avoid passing it as a parameter. The same for the baseConfig/URL. He has been moving both into the custom hook to avoid passing them as parameters.  For example [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/src/hooks/useBridgesApi/useCreateBridgeApi.ts).

Reply:
LOL.... Unless that is exactly what you have done in this PR :D (I only just noticed the file name).